### DISCUSSION TITLE
Add content of shared album to timeline of the Partner

### DISCUSSION THREAD

**Discussion by fabian1512** [immich-app/immich#14037] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I would like to share parts of my library with my children. So far, I've done this via shared albums. 
It would be nice if you could decide whether shared albums should be displayed in the timeline or not.
That way they would have a complete timeline of their own and the shared images

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#7047
